splice_machine/src/main/java/com/splicemachine/derby/utils/Vacuum.java

```diff
@@ -68,8 +68,6 @@ public Vacuum(Connection connection) throws SQLException {
 
     public void vacuumDatabase() throws SQLException{
 
-        ensurePriorTransactionsComplete();
-
```

Comment:
Vacuum still needs to wait for all active txns to complete. An long running txn may want to access a table which was created by another txn, and dropped while it is running. The table descriptor's txnId could be null or less than oldest txn id. In either case, it would be deleted by vacumme.